aft/test_functions.py

```diff
@@ -41,7 +41,6 @@ def add_one_only_int_default(a, b=3, c=4):
 class Example(object):
 
     def __init__(self, a, b, c):
-        # type: (int, float, str) -> None
```

Comment:
Why did you delete these type annotations?

Reply:
Cause there was a use of the class that didn't conform. Maybe it would be better to change it to `Any`.